formula-coroutines/src/main/java/com/instacart/formula/coroutines/FlowAction.kt

```diff
@@ -52,8 +52,8 @@ interface FlowAction<Event> : Action<Event> {
          * @param key Used to distinguish this [Action] from other actions.
          */
         inline fun <Event> fromFlow(
-            scope: CoroutineScope = MainScope(),
             key: Any?,
+            scope: CoroutineScope = MainScope(),
```

Comment:
After the multi-threading changes within the formula core, I don't think we need the main scope here anymore.

Reply:
Would you like me to change this PR to just remove this param completely?

Reply:
I've reworked it here -> https://github.com/instacart/formula/pull/373